system_tests/run_emulator.py

```diff
@@ -25,21 +25,27 @@
 
 import psutil
 
+from google.cloud.environment_vars import BIGTABLE_EMULATOR
 from google.cloud.environment_vars import GCD_DATASET
 from google.cloud.environment_vars import GCD_HOST
 from google.cloud.environment_vars import PUBSUB_EMULATOR
 from run_system_test import run_module_tests
 
 
+BIGTABLE = 'bigtable'
+DATASTORE = 'datastore'
+PUBSUB = 'pubsub'
 PACKAGE_INFO = {
-    'datastore': (GCD_DATASET, GCD_HOST),
-    'pubsub': (PUBSUB_EMULATOR,)
+    BIGTABLE: (BIGTABLE_EMULATOR,),
+    DATASTORE: (GCD_DATASET, GCD_HOST),
+    PUBSUB: (PUBSUB_EMULATOR,),
 }
 EXTRA = {
-    'datastore': ('--no-legacy',),
+    DATASTORE: ('--no-legacy',),
 }
 _DS_READY_LINE = '[datastore] Dev App Server is now running.\n'
 _PS_READY_LINE_PREFIX = '[pubsub] INFO: Server started, listening on '
+_BT_READY_LINE_PREFIX = '[bigtable] Cloud Bigtable emulator running on '
```

Comment:
It's a sign of using a different logging framework. Not at all a surprise that different teams would write the emulators in different languages.